src/shared/services/session-storage.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SessionStorageService {
+
+  /**
+   * Get an item from the session storage.
+   */
+  public getItem(key: string): string {
+    return sessionStorage.getItem(key);
+  }
+
+  /**
+   * Set an item in the session storage.
+   */
+  public setItem(key: string, value: string): void {
+    sessionStorage.setItem(key, value);
+  }
+
+  /**
+   * Remove an item from the session storage.
+   */
+  public removeItem(key: string): void {
+    sessionStorage.removeItem(key);
+  }
+}
```

Comment:
You have added a method for get, set and removeItem in the new session storage - but in this PR only use removeItem, do you need to add the other methods right now?

Reply:
They are not being used at the moment, however in the likely event that people may want to get and set session storage values I have added them to the session storage service (same as in the MC app).